ui/src/dashboards/utils/cellGetters.ts

```diff
@@ -2,19 +2,14 @@
 import {getQueryConfigAndStatus} from 'src/shared/apis'
 
 // Utils
-import replaceTemplate, {replaceInterval} from 'src/tempVars/utils/replace'
-import {getDeep} from 'src/utils/wrappers'
+import templateReplace from 'src/tempVars/utils/replace'
 
 // Constants
 import {
   UNTITLED_GRAPH,
   NEW_DEFAULT_DASHBOARD_CELL,
 } from 'src/dashboards/constants'
-import {
-  TEMP_VAR_DASHBOARD_TIME,
-  DEFAULT_DURATION_MS,
```

Comment:
Are these constants used anywhere else? Might be good to delete them from the constants file if not

Reply:
I'll check

Reply:
They seem to be used a bunch of other places.